atest/resources/TestCheckerLibrary.py

```diff
@@ -14,48 +14,61 @@
 from robot.result.model import Body, Iterations
 from robot.utils.asserts import assert_equal
 
-
-class NoSlotsKeyword(Keyword):
+class BodyGetter:
+
+    def __getitem__(self, index):
+        if isinstance(index, str):
+            index = tuple(int(i) for i in index.split(','))
+        if isinstance(index, (int, slice)):
+            return self.body[index]
+        if isinstance(index, tuple):
+            item = self
+            for i in index:
+                item = item.body[int(i)]
+            return item
+        raise TypeError(...)
+
+class NoSlotsKeyword(Keyword, BodyGetter):
```

Comment:
- Even though that `TypeError` should never happen, it would be good to have _some_ message with it. Perhaps `type(index).__name__` so that it shows what type the index had or just `repr(index)` to show the value in a format that often indicates also the type.
- An empty line missing before class.

Reply:
i took repr(index) because it it would be a structure of something, it may be interesting what the types are.